docs/contributing/adding_a_new_resource.md

```diff
@@ -61,30 +61,37 @@ To prepare your environment for running integration tests:
 ```bash
 docker run -p 5432:5432 -e POSTGRES_PASSWORD=pass -d  postgres:13.3
 # login with AWS. See all options at https://hub.cloudquery.io/providers/cloudquery/aws/latest
-gcloud auth application-default login
+export AWS_ACCESS_KEY_ID={Your AWS Access Key ID}
+export AWS_SECRET_ACCESS_KEY={Your AWS secret access key}
```

Comment:
I would say it is better to set the `AWS_PROFILE` environment variable. That way credentials can be managed in the `~/.aws/config` and can use any of credential sources like AWS_SSO, credential_process or worst case scenario hard coded credentials

Reply:
I agree that we cannot not support hard coded credentials! I think that we should be helping/pushing users to follow the best practices. I like that idea of putting the link to the docs! Here is a [good article](https://ben11kehoe.medium.com/never-put-aws-temporary-credentials-in-env-vars-or-credentials-files-theres-a-better-way-25ec45b4d73e) about this

Reply:
We can just link to this one - https://hub.cloudquery.io/providers/cloudquery/aws/latest#authentication 
This will also serve as nice cross link to our docs